packages/rancher-monitoring/rancher-monitoring.patch

```diff
@@ -1,82 +1,60 @@
-diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-monitoring/charts-original/Chart.yaml packages/rancher-monitoring/charts/Chart.yaml
+diff -x '*.tgz' -x '*.lock' -x '*.DS_Store' -uNr packages/rancher-monitoring/charts-original/Chart.yaml packages/rancher-monitoring/charts/Chart.yaml
```

Comment:
`-x ‘.DS_Store’` is a relic of the script used to generate this patch, should be removed from the patch.